tests/ca/content-addressed.nix

```diff
@@ -61,6 +61,14 @@ rec {
       echo ${rootCA}/non-ca-hello > $out/dep
     '';
   };
+  noRuntimeDep = mkDerivation {
+    name = "no-runtime-dep";
+    buildCommand = ''
+      mkdir -p $out
+      echo ${rootCA}
+      touch $out
+    '';
+  };
```

Comment:
I figure this PR also deserves a few tests adding to https://github.com/NixOS/nix/blob/d05d175ddfc3dae2e8ba33bce6935bb55773e6fa/tests/ca/build-dry.sh

Reply:
Done, found and fixed the existing test.